src/common/users.ts

```diff
@@ -18,6 +18,7 @@ export interface User {
   email: string;
   name: string;
   image: string;
+  infoConfirmed: boolean;
```

Comment:
Maybe it can be just `confirmed` or `verified`, what do you think?

Reply:
We must keep it `infoConfirmed` for backwards compatibility